pullrequests.go

```diff
@@ -347,3 +349,22 @@ func (p *PullRequests) buildPullRequestCommentBody(co *PullRequestCommentOptions
 
 	return string(data), nil
 }
+
+func buildReviewers(defaultReviewers []string) []map[string]interface{} {
+	var reviewers []map[string]interface{}
+	for _, reviewer := range defaultReviewers {
+		map1 := map[string]interface{}{
+			"approved": true,
+			"status":   "UNAPPROVED",
+			"role":     "AUTHOR",
+			"user": map[string]interface{}{
```

Comment:
Is this just initialization I guess and slug, name & displayName can be set to more specific information later?